test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/SelectExpandClauseExtensionsTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="SelectExpandClauseExtensionsTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OData.Tests.UriParser;
+using Microsoft.OData.Tests.UriParser.Binders;
+using Microsoft.OData.UriParser;
+using Microsoft.OData.Edm;
+using Xunit;
+using ODataErrorStrings = Microsoft.OData.Strings;
+using Microsoft.OData.UriParser.Aggregation;
+
+namespace Microsoft.OData.Tests
+{
+    public class SelectExpandClauseExtensionsTests
+    {
+        [Fact]
+        public void GetCurrentLevelSelectListTestMultiSelect()
+        {
+            //Arrange
+            var expected = new List<string>();
+            expected.Add("Name");
+            expected.Add("Shoe");
+
+            //Act
+            var clause = CreateSelectExpandClauseMultiSelect();
+            var actual = SelectExpandClauseExtensions.GetCurrentLevelSelectList(clause);
+
+            //Assert
+            Assert.Equal(expected.Count, actual.Count);
+            Assert.Equal(0,expected.Except(actual).Count());
+            Assert.Equal(0,actual.Except(expected).Count());

```

Comment:
why don't u compare the collection directly? or you can convert "actual" as a string, and compare the string with "expected" string?

Reply:
Used SequenceEqual, #Resolved